gcp-deployment-manager/startup.sh

```diff
@@ -20,13 +20,22 @@ PROJECT_INACTIVITY_TIMEOUT="0.5h"
 PROJECT_DANGLING_TIMEOUT="168h"
 PROJECTS_ROOT="${BUILD_USER_HOME}/builds"
 
+# run init sctipt once
+[[ -f /root/init.lock ]] && exit 0
```

Comment:
What is the harm in running startup.sh every time? Isn't the purpose of the script meant to ensure state? If it doesn't run every time, how can that happen?